scripts/create-a-m-o/functions.js

```diff
@@ -0,0 +1,18 @@
+const chalk = require('chalk');
+const outdent = require('outdent');
+//
+// const getAMOType = done => () => {
+//
+//   console.log(chalk.cyan(outdent`
+//
+//     Ok good, we will setup your repo. I have to do some checks before
+//
+//   `));
+//
+//   done();
+// };
+// 
+//
+// module.exports = {
+//   getAMOType
+// };
```

Comment:
Please either uncomment or remove this. We should not check in uncommented code. If we need to go back, we can use version control. :)

Reply:
As it says in the PR description, this is WIP. At the end i wont have commented code